### PR TITLE
Added docs

### PR DESCRIPTION
**Pull request by kurtsergey**
https://github.com/wowmaking/hybro/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,100 @@
...
+
+### Installation
+
+Install and link [react-native-webview](https://github.com/react-native-community/react-native-webview)
+
+```
+npm i react-native-webview
+react-native link react-native-webview
+```
+
```
Comment (StanislavMayorov) [inline README.md:21]: Add hybro install line

File: README.md [hunk 2]
```diff
@@ -1 +1,100 @@
...
+
+### HybroView Props
+
+* packages - object of plugged packages with names to call from web side
+
+---
+
+## Web API
+
```
Comment (StanislavMayorov) [inline README.md:63]: Please add RN API description too

File: README.md [hunk 3]
```diff
@@ -1 +1,100 @@
 # hybro
+
+ReactNative <-> Web invoke/events communication
```
Comment (StanislavMayorov) [inline README.md]:
```suggestion
ReactNative <-> WebView invoke/events communication
```

**Comment by StanislavMayorov** [comment]
thanks for quick implementation

File: README.md (hunk 2 above)
Comment (kurtsergey) [reply README.md:63]: `packages` - single property

File: README.md (hunk 1 above)
Comment (StanislavMayorov) [reply README.md:21]: I think we miss installation hybro in web app. And this part should be moved from RN part of docs to common.

File: README.md (hunk 2 above)
Comment (StanislavMayorov) [reply README.md:63]: We should add example for `EventTarget` in RN part. it's not obvious at all

**Pull request merged**